lib/tasks/mysql.rake

```diff
@@ -0,0 +1,70 @@
+namespace :mysql do
+  desc 'execute a SQL command. Usage: rake db:execute SQL=""'
+  task :execute => :environment do
+    ActiveRecord::Base.establish_connection( (ENV['ENV'] || Rails.env).to_sym)
+    ActiveRecord::Base.connection.execute( ENV['SQL'].to_s ).each { |hash| p hash }
+  end
+
+  desc 'Update values in settings table for development. Useful if you restore production database'
+  task :dev_settings => :environment do
```

Comment:
Would it potentially be a little cleaner to truncate the table and re-run the populator instead of manually assigning the values here? That way it's consistent with `defaults.json`

Reply:
Maybe, but I guess in my case I don't want the settings that are in defaults.json, I want _most_ of the settings we've configured in production.